src/wallet/spend.h

```diff
@@ -122,8 +122,8 @@ bool AttemptSelection(const CWallet& wallet, const CAmount& nTargetValue, const
  * param@[out]  nValueRet           Total value of selected coins including pre-selected ones
  *                                  from coin_control and Coin Selection if successful.
  */
-bool SelectCoins(const CWallet& wallet, const std::vector<COutput>& vAvailableCoins, const CAmount& nTargetValue, std::set<CInputCoin>& setCoinsRet, CAmount& nValueRet,
-                 const CCoinControl& coin_control, CoinSelectionParams& coin_selection_params) EXCLUSIVE_LOCKS_REQUIRED(wallet.cs_wallet);
+std::optional<SelectionResult> SelectCoins(const CWallet& wallet, const std::vector<COutput>& vAvailableCoins, const CAmount& nTargetValue, const CCoinControl& coin_control,
+                 CoinSelectionParams& coin_selection_params) EXCLUSIVE_LOCKS_REQUIRED(wallet.cs_wallet);
```

Comment:
- It looks like `CoinsSelectionParams&` can be const
- Doc comment above needs to be updated, it refers to arguments that no longer exist

Reply:
Done both.